charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -149,9 +149,9 @@ spec:
           - name: DD_ADMISSION_CONTROLLER_SERVICE_NAME
             value: {{ template "datadog.fullname" . }}-cluster-agent-admission-controller
           {{- end }}
-          {{- if .Values.datadog.clusterChecks.enabled }}
+          {{- if (eq (include "should-enable-cluster-check-workers" .) "true") }}
```

Comment:
I don't think we should do this, because we can run cluster checks without cluster check runners. This change will disable the checks for users who use node agents to run cluster checks.